moc-health-check/config.cfg

```diff
@@ -0,0 +1,22 @@
+[ConfigInfo]
+
+# Size of the file that is sent to Pfioh in MB (1,2,3...)
+SIZE=6
+
+# Number of attempts required to check success rate
+RANGE=2
+
+# Number of seconds to wait before ending a non-responding Pfurl command
+TIMEOUT=8
+
+# Success rate when program should email
+THRESHOLD=70
+
+#Number of seconds to wait for job to execute before checking status of Pman (base value for exponential backoff)

```

Comment:
Space after #

Reply:
Also given it's an initial value for exponential back backoff, would a name like INITIAL_WAIT be better?